dotnet/src/Connectors/Connectors.Memory.PgVector/PostgresVectorStore.cs

```diff
@@ -32,7 +32,7 @@ public sealed class PostgresVectorStore : VectorStore, IDisposable
     /// Initializes a new instance of the <see cref="PostgresVectorStore"/> class.
     /// </summary>
     /// <param name="dataSource">Postgres data source.</param>
-    /// <param name="ownsDataSource">A value indicating whether the data source should be disposed after the vector store is disposed.</param>
+    /// <param name="ownsDataSource">A value indicating whether the data source must be disposed after the vector store is disposed.</param>
```

Comment:
I don't think "must" or "should" are appropriate here: we're just documenting the behavior of this type (PostgresVectorStore).

```suggestion
    /// <param name="ownsDataSource">A value indicating whether <paramref name="dataSource"/> is disposed when this instance of <see cref="PostgresVectorStore"/> is disposed.</param>
```

(also for the other providers etc.)